cubitpy/cubitpy_types.py

```diff
@@ -165,18 +165,15 @@ def get_baci_name(self):
         """Get the name of this element in baci."""
 
         # Get the element type parameters.
-        if self == self.hex8:
-            return "SOLIDH8"
-        elif self == self.hex20:
-            return "SOLIDH20"
-        elif self == self.hex27:
-            return "SOLIDH27"
-        elif self == self.tet4:
-            return "SOLIDT4"
-        elif self == self.tet10:
-            return "SOLIDT10"
-        elif self == self.hex8sh:
-            return "SOLIDSH8"
+        if (
+            self == self.hex8
+            or self == self.hex20
+            or self == self.hex27
+            or self == self.tet4
+            or self == self.hex8sh
+            or self == self.tet10
+        ):
+            return "SOLID"
```

Comment:
The old/new solid implementation is a little bit more complex as can be covered by this simple change. Let's merge the tutorial and talk about how to best implement this. For example, the `self.hex8sh` is a special solid shell element that is not yet converted by the new solid element implementation, also we might want to keep the option to create "old" solid elements until this issue is fully resolved in BACI.

Reply:
Reverted as discussed